ledger/participant-integration-api/src/main/scala/platform/index/ReadOnlySqlLedgerWithMutableCache.scala

```diff
@@ -83,6 +87,7 @@ private[index] object ReadOnlySqlLedgerWithMutableCache {
         cacheUpdatesDispatcher: Dispatcher[(Offset, Long)],
         generalDispatcher: Dispatcher[Offset],
         dispatcherLagMeter: DispatcherLagMeter,
+        startExclusive: (Offset, EventSequentialId),
```

Comment:
In this file we interchangeably use `(Offset, EventSequentialId)` and `(Offset, Long)` for offset-eventSeqId. Could we unify it?

Reply:
Reverted both new usages to `Long` to minimize changes